### PR TITLE
Feature/path following perception

### PR DESCRIPTION
**Pull request by ivangushkov**
Pretty much ready for pool test.

### CODE REVIEW

File: path_following/CMakeLists.txt [hunk 1]
```diff
@@ -0,0 +1,197 @@
+cmake_minimum_required(VERSION 3.0.2)
```
Comment (chrstrom) [inline path_following/CMakeLists.txt:1]: Remove commented lines in this file

**Review by chrstrom — CHANGES_REQUESTED** [review]
Very minor comment cleanup. After that, we can merge if this worked well on the test data. @ivangushkov, what is the status on that?

File: path_following/CMakeLists.txt (hunk 1 above)
Comment (ivangushkov) [reply path_following/CMakeLists.txt:1]: Yessir!

**Review by theBadMusician — APPROVED** [review]
LGTM

**Pull request merged**